.gitignore

```diff
@@ -35,3 +35,6 @@ ci_version.json
 
 # Why though? I don't know
 /logs/latest.log
+.idea/codeStyles/Project.xml
```

Comment:
what happened to that file? This file should not have been changed.

Reply:
i have no clue, probably idea jank

Reply:
Would be a good idea to revert those changes then.